site/content/docs/main/locations.md

```diff
@@ -16,12 +16,17 @@ The user can pre-configure one or more possible `BackupStorageLocations` and one
 This configuration design enables a number of different use cases, including:
 
 - Take snapshots of more than one kind of persistent volume in a single Velero backup. For example, in a cluster with both EBS volumes and Portworx volumes
-- Have some Velero backups go to a bucket in an eastern USA region, and others go to a bucket in a western USA region
+- Have some Velero backups go to a bucket in an eastern USA region, and others go to a bucket in a western USA region, or to a different storage provider
 - For volume providers that support it, like Portworx, you can have some snapshots stored locally on the cluster and have others stored in the cloud
 
 ## Limitations / Caveats
 
-- Velero only supports a single set of credentials *per provider*. It's not yet possible to use different credentials for different locations, if they're for the same provider.
+- Velero supports multiple credentials for `BackupStorageLocations`, allowing you to specify the credentials to use with any `BackupStorageLocation`.
+  However, use of this feature requires support from object storage provider plugin.
```

Comment:
i'm not the clearest on the plugin breakdown, so please feel free to ignore me, but to me this reads a little confusingly. Is object storage provider plugin a feature of the plugins Velero supports? or how we refer to the plugins themselves?

Reply:
This confused me for quite a while too. Velero has different [plugin kinds](https://velero.io/docs/v1.5/custom-plugins/#plugin-kinds) and each of the plugin provider repos can provide multiple plugins. For example, if you look at the overview of plugins, you can see labels for which plugin kinds are available in each repo: https://velero.io/plugins/

I can definitely rephrase this sentence though to try and make it clearer :+1: